tasks/agent6.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Create main Datadog agant yaml configuration file (beta)
+  template:
+    src: datadog.yaml.j2
+    dest: /etc/datadog-agent/datadog.yaml
+    owner: '{{ datadog_user }}'
+    group: '{{ datadog_group }}'
+  notify: restart datadog-agent
+
+- name: Creating configuration files for extra
```

Comment:
Missed this on first review: this file won't be read by the external agents by default unfortunately

Currently we have to write a conf file for each of the external agents (trace and process), c.f. what's done in https://github.com/DataDog/chef-datadog/blob/master/recipes/_agent6_config.rb#L22-L105

A simple and dirty way to do this would be to use the same approach as the chef cookbook, i.e. simply drop these 2 conf files in their correct default locations, with the contents of a full agent5-style `datadog.conf`.

Let me know if you have any questions... It's a bit hacky but we should be able to drop it as soon as these agents can read from `datadog.yaml`

Reply:
:+1: